src/mappings/modules/liquidation/clipper.ts

```diff
@@ -5,9 +5,50 @@ import {
   Take as TakeEvent,
   Yank as YankEvent,
   Redo as RedoEvent,
+  File as FileBigIntEvent,
+  File1 as FileAddressEvent,
```

Comment:
It would be better to have a more memotecnic name

Reply:
I think the naming is ok here.